extract/ext/office_extract.py

```diff
@@ -9,6 +9,7 @@
 import math
 import binascii
 import tempfile
+import msoffcrypto
```

Comment:
We were already running a msoffice-crypt.exe on office documents (https://github.com/CybercentreCanada/assemblyline-service-extract/blob/ac3ea1286341de687f16da34ffa07cc264a6b6cf/extract/ext/doc_extract.py#L436).

Are we sure that this is not just adding the same thing twice? Someone needs to test  msoffice-crypt.exe vs the msoffcrypto lib before merging this in. We need to know which is faster, which extract more documents and if we need to run both.